js/coinsbit.js

```diff
@@ -0,0 +1,742 @@
+                // ATM, they need to call separately (unfortunately)
+                request['side'] = this.safeString (params, 'side', 'buy');
+                const requestBids = this.spotPublicGetBook (this.extend (request, params));
+                request['side'] = this.safeString (params, 'side', 'sell');
+                const requestAsks = this.spotPublicGetBook (this.extend (request, params));
+                const responses = await Promise.all ([requestBids, requestAsks]);
+                // FOR EACH (BUY/SELL) REQUEST, THE RESPONSE IS SIMILAR (JUST 'side' PROPERTY IS DIFFERENT BETWEEN THEM)
+                //     {
+                //         success: true,
+                //         message: "",
+                //         result: {
+                //             offset: 0,
+                //             limit: 2,
+                //             total: 334,
+                //             orders: [
+                //                 {
+                //                     id: 8620142523,
+                //                     left: "1.56772",
+                //                     market: "BTC_USDT",
+                //                     amount: "1.56772",
+                //                     type: "limit",
+                //                     price: "46752.63032535",
+                //                     timestamp: 1640031161.544,
+                //                     side: "buy",
+                //                     dealFee: "0",
+                //                     takerFee: "0",
+                //                     makerFee: "0",
+                //                     dealStock: "0",
+                //                     dealMoney: "0"
+                //                 },
+                //                 ...
+                //             ]
+                //         },
+                //         code: 200
+                //     }
+                const resultBids = this.safeValue (responses[0], 'result', {});
+                const resultAsks = this.safeValue (responses[1], 'result', {});
+                const bids = this.safeValue (resultBids, 'orders', []);
+                const asks = this.safeValue (resultAsks, 'orders', []);
+                const response = { 'bids': bids, 'asks': asks };
+                const priceKey = 'price';
+                const amountKey = 'amount';
+                const timestamp = this.milliseconds ();
+                orderbook = this.parseOrderBook (response, symbol, timestamp, 'bids', 'asks', priceKey, amountKey);
+            }
+        }
+        // orderbook['nonce']  <<< TODO >>> -- atm I dont know how to handle that, as the response doesn't have right that property
+        return orderbook;
+    }
+
+    async fetchOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
+        const request = this.handleMarketTypeAndParams ('fetchOHLCV', undefined, params)[1];
```

Comment:
`handleMarketTypeAndParams` doesn't actually return a request-like object, check how this is done in the existing exchanges.

The usage for `handleMarketTypeAndParams` is always:

```JavaScript
await this.loadMarkets (); // always load the markets before accessing the markets
const market = this.market (symbol);
[ type, params ] = this.handleMarketTypeAndParams ('fetchOHLCV', market, params);
```

Reply:
done